Lib/glyphsLib/parser.py

```diff
@@ -98,14 +100,39 @@ def _parse_dict_into_object(self, res, d):
                 res[name] = d[name]
 
 
-def load(fp):
-    """Read a .glyphs file. 'fp' should be (readable) file object.
+def load_glyphspackage(package_dir):
+    infofile = os.path.join(package_dir, "fontinfo.plist")
+    orderfile = os.path.join(package_dir, "order.plist")
+    glyphorder = openstep_plist.load(open(orderfile, "r", encoding="utf-8"))
+    data = openstep_plist.load(open(infofile, "r", encoding="utf-8"), use_numbers=True)
+    data["glyphs"] = []
+    for g in glyphorder:
+        glyphname = userNameToFileName(g) + ".glyph"
+        glyphfile = os.path.join(package_dir, "glyphs", glyphname)
+        data["glyphs"].append(
+            openstep_plist.load(open(glyphfile, "r"), use_numbers=True)
+        )
+
+    return data
+
+
+def load(file):
```

Comment:
nit: "path" can be understood as either a file name or a directory name, whereas "file" is usually just a single file name or the file object

```suggestion
def load(file_or_path):
```